dm/config/task.go

```diff
@@ -305,6 +306,8 @@ type TaskConfig struct {
 	Syncers   map[string]*SyncerConfig   `yaml:"syncers"`
 
 	CleanDumpFile bool `yaml:"clean-dump-file"`
+
+	EnableANSIQuotes bool `yaml:"ansi-quotes" toml:"ansi-quotes" json:"ansi-quotes"`
```

Comment:
rename to `ansi-quotes` to avoid conflict with deprecated sycncer config `enable-ansi-quotes`